source/extensions/tracers/opentelemetry/http_trace_exporter.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "envoy/config/core/v3/http_service.pb.h"
+#include "envoy/upstream/cluster_manager.h"
+
+#include "source/common/common/logger.h"
+#include "source/common/http/async_client_impl.h"
+#include "source/common/http/async_client_utility.h"
+#include "source/common/http/headers.h"
+#include "source/common/http/message_impl.h"
+#include "source/common/http/utility.h"
+
+#include "opentelemetry/proto/collector/trace/v1/trace_service.pb.h"
+#include "trace_exporter.h"
+#include "tracer_stats.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace OpenTelemetry {
+
+/**
+ * Exporter for OTLP traces over HTTP.
+ */
+class OpenTelemetryHttpTraceExporter : public OpenTelemetryTraceExporter,
+                                       public Http::AsyncClient::Callbacks {
+public:
+  OpenTelemetryHttpTraceExporter(Upstream::ClusterManager& cluster_manager,
+                                 envoy::config::core::v3::HttpService http_service,
+                                 OpenTelemetryTracerStats& tracing_stats);
+
+  // The default path to use when OpenTelemetryConfig::HttpConfig::traces_path is empty
+  const Http::LowerCaseString TRACES_PATH{"/v1/traces"};
```

Comment:
Is this being used somewhere?

Reply:
Ah no, not anymore. Removed in: [2a5d544](https://github.com/envoyproxy/envoy/pull/29207/commits/2a5d5448406e99fbfaecda651e0d176eda96383a)